app/boards/shields/helix_4/Kconfig.defconfig

```diff
@@ -0,0 +1,37 @@
+
+if SHIELD_HELIX_4_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "Helix"
+
+config ZMK_SPLIT_BLE_ROLE_CENTRAL
+	default y
+
+endif
+
+if SHIELD_HELIX_4_RIGHT
+
+config ZMK_KEYBOARD_NAME
+	default "Helix Right"
+
+config USB
+	default y
+
+endif
+
+if SHIELD_HELIX_4_LEFT || SHIELD_HELIX_4_RIGHT
+
+config ZMK_SPLIT
+  default y
+
+config DEVICE_POWER_MANAGEMENT
+	default y
```

Comment:
This is already enabled in the main Kconfig I think? Should be able to remove it.

Reply:
I wasn't sure. When I checked the full config output it was not set, and `app/Kconfig` has it under `ZMK_SLEEP`. Is that the one that should be `default y`?